components/webapps/operator-onboard/src/main/java/com/wso2telco/dep/service/OperatorService.java

```diff
@@ -36,4 +39,33 @@ public Response addOperator(String jsonBody) throws Exception{
 		
 		return null;	
 	}
+
+	public void fooMethod(String abc) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "abc". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.wso2telco.com:9000/coding_rules#rule_key=squid%3AS1172)